developer/src/tike/debug/Keyman.System.Debug.DebugUtils.pas

```diff
@@ -19,18 +19,20 @@ function GetContextFromMemo(Memo: TKeymanDeveloperDebuggerMemo; DeadKeys: TDebug
   n, i: Integer;
   ch: Char;
   dk: TDeadKeyInfo;
+  t: string;
 begin
   n := 0;
-  SetLength(Result, Length(Memo.Text)+1);
+  t := Memo.GetTextCR;
+  SetLength(Result, t.Length+1);
```

Comment:
CI failed with
```
debug\Keyman.System.Debug.DebugUtils.pas(26): error E2018: Record, object or class type required 
debug\Keyman.System.Debug.DebugUtils.pas(31): error E2018: Record, object or class type required 
child\UfrmDebug.pas(37): error F2063: Could not compile used unit 'Keyman.System.Debug.DebugUtils.pas' 
```

Is this?
```suggestion
  SetLength(Result, Length(t) + 1);
```

Reply:
I've fixed this by using System.SysUtils (missed from the selective commit)